packages/@react-aria/collections/src/Hidden.tsx

```diff
@@ -35,7 +35,7 @@ if (typeof HTMLTemplateElement !== 'undefined') {
 
 export const HiddenContext = createContext<boolean>(false);
 
-export function Hidden(props: {children: ReactNode}): ReactNode {
+export function Hidden(props: {children: ReactNode}): JSX.Element {
```

Comment:
Why JSX.Element here but ReactElement in other places?

Reply:
For some of these, unpkg was down, so I relied on the ts inference
this one is internal, so doesn't matter too much

Reply:
sure, though it may have been inconsistent before as well. Just wondering which one we _should_ use.

Reply:
I've double checked all the old ones and included direct links to the previously generated output

Reply:
Including link to comment on type https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d2c29760d5939b14a41084d0154efbf45161bcfe/types/react/index.d.ts#L4005